package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-middlewares",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
do you mean that I can't edit the minor digit in the semver? is it restricted to adding middleware only?
is it okay if I change the last digit? "2.1.1" instead of "2.2.0"?